docs/API.md

```diff
@@ -2501,13 +2507,28 @@ The response holds a JSON object containing information about the upload.
 
 ```go
 type UploadInfoResponse struct {
```

Comment:
Perhaps just refer to the POST /upload endpoint rather than duplicating the type definition?